FRW/Modules/Wallet/TokenDetail/TokenDetailViewModel.swift

```diff
@@ -168,6 +168,8 @@ class TokenDetailViewModel: ObservableObject {
     var showSheet: Bool = false
     var buttonAction: TokenDetailViewModel.Action = .none
 
+    var isTokenDetailsButtonEnabled: Bool { self.token.website != nil }
```

Comment:
Please also check the website is not empty string.

Reply:
Website is an `URL`, not a string. Not sure if being it an empty string is possible, but I've added an explicit check to make sure the absolute URL in string format is not empty